### DISCUSSION TITLE
Is this in a functional state again? if so I can bug test the master branch somewhat again.

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#35] [General]
Title explains all, also the meaning of the universe is 42.

**Comment by zer0Kerbal** [comment]
6x7
Mostly functional - at least on my local branch. including being localized. when I find time I will update this repo.

**Comment by zer0Kerbal** [comment]
Use the 1.3.99.0-adoption branch - it is the current work branch. I won't be merging into master until ready to release.

**Reply by FuryFire2004** [reply]
Yup will bug test crap and piss. when I get HOME.
